pkg/ruler/compat.go

```diff
@@ -199,12 +222,24 @@ func DefaultTenantManagerFactory(cfg Config, p Pusher, q storage.Queryable, engi
 		Name: "cortex_ruler_queries_failed_total",
 		Help: "Number of failed queries by ruler.",
 	})
+	var rulerQuerySeconds *prometheus.CounterVec
+	if cfg.EnableQueryStats {
+		rulerQuerySeconds = promauto.With(reg).NewCounterVec(prometheus.CounterOpts{
+			Name: "cortex_ruler_query_seconds_total",
+			Help: "Total amount of wall clock time spent processing queries by the ruler.",
+		}, []string{"user"})
```

Comment:
Right now this function will log an info level line that looks like ` level=info ts=2021-07-01T19:26:16.752146964Z caller=compat.go:180 org_id=fake traceID=189b319111e600ae msg="ruler query stats" query_wall_time_seconds=0.004081708 count(up)>1=(MISSING)` should I add the userID to this log as well or is the org_id good enough?